app/models/miq_group.rb

```diff
@@ -254,6 +254,13 @@ def self.non_tenant_groups
     where.not(:group_type => TENANT_GROUP)
   end
 
+  def self.valid_filters?(filters_hash)
+    return true  unless filters_hash                  # nil ok
+    return false unless filters_hash.kind_of?(Hash)   # must be Hash
+    return true  if filters_hash.blank?               # {} ok
```

Comment:
ya spent some time here, I needed those seperated as to differentiate with [] which is also considered blank.